### PR TITLE
Added bootstrap library

### PR DESCRIPTION
**Pull request by r10a**
Added bootstrap library as part of Issue #2

### CODE REVIEW

**Comment by r10a** [comment]
Have included bootstrap library in vendor js and css.

**Review by viniciusmeneses — CHANGES_REQUESTED** [review]
Awesome! Fix the following problems described below and I'll merge it.

File: src/assets/scss/vendor/index.scss [hunk 1]
```diff
@@ -1,2 +1,3 @@
 @import './helper';
 @import './normalize';
+@import './bootstrap.scss'
```
Comment (viniciusmeneses) [inline src/assets/scss/vendor/index.scss]: Remove `.scss` to be equal to other imports.

File: package.json [hunk 2]
```diff
@@ -34,5 +34,7 @@
     "lodash": "^4.17.11",
     "minimatch": "^3.0.4"
   },
-  "dependencies": {}
+  "dependencies": {
+    "bootstrap": "^4.1.3"
```
Comment (viniciusmeneses) [inline package.json]: You don't need install Boostrap as dependency if you don't import it through node_modules, so remove it please :)

**Review by viniciusmeneses — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (r10a) [reply package.json]: Sure. Thanks for the review. I will get it done quick

**Comment by r10a** [comment]
Hey. I have made the changes you asked.

**Review by viniciusmeneses — APPROVED** [review]

**Comment by viniciusmeneses** [comment]
Thanks for your contribution!

**Pull request merged**